### ISSUE TITLE
Decide whether AdditionalAnnouncement should be ON by default

### ISSUE THREAD

**Issue by zuevval** [braille-systems/learn-braille#230]
- [x] ask in WhatsApp
- [x] fix `dev` if answer is 'yes'
- [x] make setting description more accurate (mention that tasks titles announcement is affected, too)

this rose up due to: https://github.com/braille-systems/learn-braille/commit/77d1cc2f426bb9c7c11b109152594e0092a65c36#commitcomment-41042668
